snapcraft.yaml

```diff
@@ -18,7 +18,7 @@ summary: Mozilla Firefox web browser
 description:  Firefox is a powerful, extensible web browser with support for modern web application technologies.
 confinement: strict
 grade: stable
-base: core22
+base: core24
```

Comment:
Are we officially moving to core24 by default ?

Reply:
Not yet.

Reply:
As far as I can see, we'll only build Firefox on RISC if it is on the
24.04 base.

22.04 does not have a recent enough tool-chain. In particular, LLVM 18
missing from 22.04 was acknowledged in
https://launchpad.net/bugs/2075477. If that were to be fixed, there are
good chances we can keep it on core22, as the version of Rust needed
(1.82) is already on 22.04.

Reply:
Right, but you are not only moving the riscV build to core24 but every other arch with it, right?
We have core24 builds, rather than changing beta maybe we should just manually publish builds from the core24 setup for riscV?